README.md

```diff
@@ -1 +1,50 @@
-# landing-page
\ No newline at end of file
+# Landing Page
+
+This project is about review on existing project which is completed during admission process at Microverse. And also comply the following things,
+
+- Apply best practices in HTML code.
+- Use semantic HTML tags.
+- Use CSS selectors correctly.
+- Use CSS box model.
+- Perform a code review for yourself on a past project.
+
+
+## Build with
+- HTML
+- CSS
+
+## Live Demo
+There is no live demo for this project not.
```

Comment:
It is very important you write a well-documented readme in order to make your repo look more professional. I'd suggest you use this [readme](https://github.com/microverseinc/readme-template/blob/master/README.md) template from Microverse 😄

Reply:
- Hello @developertariq , This is GREATTT!. Keep rocking 🚀 🚀🚀🚀